### PR TITLE
[Build] Reuse pom.xml and build.xml elements to simplify build

### PR DESCRIPTION
**Pull request by HannesWell**
Simplify the build by re-using only one build.xml and generalizing the pom.xml files.

It could even be consider to fully inline the build.xml file into the antrun-plugin target.

### CODE REVIEW

File: bundles/binaries-parent/build.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project name="org.eclipse.swt.cocoa.macosx.x86_64" default="build.jars" basedir=".">
+<project name="org.eclipse.swt.${ws}.${os}.${arch}" default="build.jars" basedir=".">
```
Comment (HannesWell) [inline bundles/binaries-parent/build.xml:2]: I wonder if the name has to match the platform or if this is irrelevant and could just have any constant name?

File: pom.xml [hunk 2]
```diff
@@ -91,8 +91,7 @@
         <groupId>org.eclipse.tycho</groupId>
         <artifactId>tycho-compiler-plugin</artifactId>
         <configuration>
-          <source>11</source>
-          <target>11</target>
+          <release>11</release>
```
Comment (HannesWell) [inline pom.xml:94]: Is there a reason the `release` property was not used before?

**Comment by HannesWell** [comment]
The test failure also happens on the master branch.

File: pom.xml (hunk 2 above)
Comment (akurtakov) [reply pom.xml:94]: The only reason is that when we created that there was no release property :)

File: bundles/binaries-parent/build.xml (hunk 1 above)
Comment (akurtakov) [reply bundles/binaries-parent/build.xml:2]: Ant's project name shouldn't matter but let's have it different for the case when we have to debug smth.

**Comment by akurtakov** [comment]
Looks very good to me.

**Comment by HannesWell** [comment]
> Looks very good to me.

Thanks for reviewing.

**Pull request merged**